app/views/episodes/_form_status.html.erb

```diff
@@ -47,6 +47,22 @@
         </p>
       </div>
     <% end %>
+    <% if episode.persisted? && episode.publish_to_integration?(:megaphone) %>
+      <% integration = :megaphone %>
+      <% integration_status = episode_integration_status(integration, episode) %>
+      <div class="col-12 mt-4">
+        <p class="status-text">
+          <strong><%= "#{integration.to_s.titleize} Status" %>:</strong>
```

Comment:
Nitpick: why not just translate the whole thing?  It looks like you were trying to make this generic for any integration, but didn't update the Apple one above this.

Reply:
yeah, I was trying not to mess too much with Apple code when I didn't need to, but put in what I thought would be a way to make this code reusable for both used only for megaphone.
I guess I could go ahead and refactor....